api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImplTest.java

```diff
@@ -394,4 +412,75 @@ public void searchForObs_shouldHandleComplexQuery() {
 			assertThat(resultsList.get(i - 1).getObsDatetime(), sameOrAfter(resultsList.get(i).getObsDatetime()));
 		}
 	}
+	
+	@Test
+	public void searchForObs_shouldReturnObsByMemberReference() {
+		ReferenceParam memberReference = new ReferenceParam();
+		memberReference.setValue(MEMBER_UUID);
+		
+		Collection<Obs> results = dao.searchForObservations(null, memberReference, null, null, null, null, null, null, null,
+		    null);
+		
+		assertThat(results, notNullValue());
+		assertThat(results, not(empty()));
+		assertThat(results.iterator().next().getUuid(), equalTo(OBS_UUID));
+	}
+	
+	@Test
+	public void searchForObs_shouldSearchForObsByValueConceptId() {
+		TokenAndListParam code = new TokenAndListParam();
+		TokenParam codingToken = new TokenParam();
+		codingToken.setValue(VALUE_CONCEPT_ID);
+		code.addAnd(codingToken);
+		
+		Collection<Obs> results = dao.searchForObservations(null, null, null, code, null, null, null, null, null, null);
+		
+		assertThat(results, notNullValue());
+		assertThat(results, not(empty()));
+		assertThat(results.iterator().next().getUuid(), equalTo(OBS_VALUE_CONCEPT_UUID));
+	}
+	
+	@Test
+	public void searchForObs_shouldSearchForObsByValueDate() {
+		Collection<Obs> results = dao.searchForObservations(null, null, null, null,
+		    new DateRangeParam(new DateParam(VALUE_DATE_AND_TIME)), null, null, null, null, null);
+		
+		assertThat(results, notNullValue());
+		assertThat(results, not(empty()));
+		assertThat(results, hasItem(hasProperty("uuid", equalTo(OBS_UUID))));
+	}
+	
+	//	@Test
+	//	public void searchForObs_shouldSearchForObsByValueQuantity() {
+	//		QuantityAndListParam quantityAndListParam=new QuantityAndListParam();
+	//
```

Comment:
@ibacher Please help me resolve this issue

Reply:
Change `VALUE_QUANTITY` from "134.0" to 134.0d and change the type to double.

Reply:
@ibacher Still the same error

Reply:
@VaishSiddharth Can you put the full stack trace somewhere?

Reply:
``` java
@Test
	public void searchForObs_shouldSearchForObsByValueQuantity() {
		QuantityAndListParam quantityAndListParam = new QuantityAndListParam();
		
		QuantityOrListParam quantityOrListParam = new QuantityOrListParam();
		
		QuantityParam quantityParam = new QuantityParam();
		quantityParam.setValue(134.0d);
		quantityAndListParam.addAnd(quantityOrListParam.add(quantityParam));
		
		// Error java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Double (java.lang.String and java.lang.Double are in module java.base of loader 'bootstrap')
		Collection<Obs> results = dao.searchForObservations(null, null, null, null, null, quantityAndListParam, null, null,
		    null, null);
		
		assertThat(results, notNullValue());
		assertThat(results, not(empty()));
		assertThat(results, hasItem(hasProperty("uuid", equalTo(OBS_UUID))));
	}
```
```
/usr/lib/jvm/jdk-13.0.1/bin/java -ea -Didea.test.cyclic.buffer.size=1048576 -javaagent:/snap/intellij-idea-community/208/lib/idea_rt.jar=45755:/snap/intellij-idea-community/208/bin -Dfile.encoding=UTF-8 -classpath /snap/intellij-idea-community/208/lib/idea_rt.jar:/snap/intellij-idea-community/208/plugins/junit/lib/junit5-rt.jar:/snap/intellij-idea-community/208/plugins/junit/lib/junit-rt.jar:/home/sid/IdeaProjects/openmrs-module-fhir2/api/target/test-classes:/home/sid/IdeaProjects/openmrs-module-fhir2/api/target/classes:/home/sid/.m2/repository/ca/uhn/hapi/fhir/hapi-fhir-base/4.2.0/hapi-fhir-base-4.2.0.jar:/home/sid/.m2/repository/com/google/code/gson/gson/2.8.5/gson-2.8.5.jar:/home/sid/.m2/repository/org/apache/commons/commons-lang3/3.9/commons-lang3-3.9.jar:/home/sid/.m2/repository/org/apache/commons/commons-text/1.7/commons-text-1.7.jar:/home/sid/.m2/repository/commons-codec/commons-codec/1.12/commons-codec-1.12.jar:/home/sid/.m2/repository/ca/uhn/hapi/fhir/hapi-fhir-structures-r4/4.2.0/hapi-fhir-structures-r4-4.2.0.jar:/home/sid/.m2/repository/ca/uhn/hapi/fhir/org.hl7.fhir.utilities/4.2.0/org.hl7.fhir.utilities-4.2.0.jar:/home/sid/.m2/repository/ca/uhn/hapi/fhir/org.hl7.fhir.r4/4.2.0/org.hl7.fhir.r4-4.2.0.jar:/home/sid/.m2/repository/org/fhir/ucum/1.0.2/ucum-1.0.2.jar:/home/sid/.m2/repository/xpp3/xpp3/1.1.4c/xpp3-1.1.4c.jar:/home/sid/.m2/repository/xpp3/xpp3_xpath/1.1.4c/xpp3_xpath-1.1.4c.jar:/home/sid/.m2/repository/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar:/home/sid/.m2/repository/org/projectlombok/lombok/1.18.10/lombok-1.18.10.jar:/home/sid/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar:/home/sid/.m2/repository/com/google/guava/guava/28.1-jre/guava-28.1-jre.jar:/home/sid/.m2/repository/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar:/home/sid/.m2/repository/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/sid/.m2/repository/org/checkerframework/checker-qual/2.8.1/checker-qual-2.8.1.jar:/home/sid/.m2/repository/com/google/errorprone/error_prone_annotations/2.3.2/error_prone_annotations-2.3.2.jar:/home/sid/.m2/repository/com/google/j2objc/j2objc-annotations/1.3/j2objc-annotations-1.3.jar:/home/sid/.m2/repository/org/codehaus/mojo/animal-sniffer-annotations/1.18/animal-sniffer-annotations-1.18.jar:/home/sid/.m2/repository/org/exparity/hamcrest-date/2.0.7/hamcrest-date-2.0.7.jar:/home/sid/.m2/repository/org/openmrs/api/openmrs-api/2.0.5/openmrs-api-2.0.5.jar:/home/sid/.m2/repository/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar:/home/sid/.m2/repository/commons-beanutils/commons-beanutils/1.7.0/commons-beanutils-1.7.0.jar:/home/sid/.m2/repository/commons-io/commons-io/1.4/commons-io-1.4.jar:/home/sid/.m2/repository/org/azeckoski/reflectutils/0.9.14/reflectutils-0.9.14.jar:/home/sid/.m2/repository/org/apache/velocity/velocity/1.6.2/velocity-1.6.2.jar:/home/sid/.m2/repository/commons-lang/commons-lang/2.4/commons-lang-2.4.jar:/home/sid/.m2/repository/log4j/log4j/1.2.15/log4j-1.2.15.jar:/home/sid/.m2/repository/org/springframework/spring-core/4.1.4.RELEASE/spring-core-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-beans/4.1.4.RELEASE/spring-beans-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-context/4.1.4.RELEASE/spring-context-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-expression/4.1.4.RELEASE/spring-expression-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-aop/4.1.4.RELEASE/spring-aop-4.1.4.RELEASE.jar:/home/sid/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar:/home/sid/.m2/repository/org/springframework/spring-orm/4.1.4.RELEASE/spring-orm-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-tx/4.1.4.RELEASE/spring-tx-4.1.4.RELEASE.jar:/home/sid/.m2/repository/org/springframework/spring-jdbc/4.1.4.RELEASE/spring-jdbc-4.1.4.RELEASE.jar:/home/sid/.m2/repository/asm/asm-commons/2.2.3/asm-commons-2.2.3.jar:/home/sid/.m2/repository/asm/asm-tree/2.2.3/asm-tree-2.2.3.jar:/home/sid/.m2/repository/asm/asm/2.2.3/asm-2.2.3.jar:/home/sid/.m2/repository/asm/asm-util/2.2.3/asm-util-2.2.3.jar:/home/sid/.m2/repository/cglib/cglib-nodep/2.2/cglib-nodep-2.2.jar:/home/sid/.m2/repository/ca/uhn/hapi/hapi-base/2.0/hapi-base-2.0.jar:/home/sid/.m2/repository/commons-cli/commons-cli/1.2/commons-cli-1.2.jar:/home/sid/.m2/repository/org/jdom/jdom/1.1/jdom-1.1.jar:/home/sid/.m2/repository/xalan/xalan/2.7.0/xalan-2.7.0.jar:/home/sid/.m2/repository/ca/uhn/hapi/hapi-structures-v25/2.0/hapi-structures-v25-2.0.jar:/home/sid/.m2/repository/ca/uhn/hapi/hapi-structures-v26/2.0/hapi-structures-v26-2.0.jar:/home/sid/.m2/repository/org/javassist/javassist/3.19.0-GA/javassist-3.19.0-GA.jar:/home/sid/.m2/repository/org/hibernate/hibernate-core/4.3.9.Final/hibernate-core-4.3.9.Final.jar:/home/sid/.m2/repository/org/jboss/logging/jboss-logging/3.1.3.GA/jboss-logging-3.1.3.GA.jar:/home/sid/.m2/repository/org/jboss/logging/jboss-logging-annotations/1.2.0.Beta1/jboss-logging-annotations-1.2.0.Beta1.jar:/home/sid/.m2/repository/org/jboss/spec/javax/transaction/jboss-transaction-api_1.2_spec/1.0.0.Final/jboss-transaction-api_1.2_spec-1.0.0.Final.jar:/home/sid/.m2/repository/org/hibernate/common/hibernate-commons-annotations/4.0.5.Final/hibernate-commons-annotations-4.0.5.Final.jar:/home/sid/.m2/repository/org/hibernate/javax/persistence/hibernate-jpa-2.1-api/1.0.0.Final/hibernate-jpa-2.1-api-1.0.0.Final.jar:/home/sid/.m2/repository/antlr/antlr/2.7.7/antlr-2.7.7.jar:/home/sid/.m2/repository/org/jboss/jandex/1.1.0.Final/jandex-1.1.0.Final.jar:/home/sid/.m2/repository/org/hibernate/hibernate-c3p0/4.3.9.Final/hibernate-c3p0-4.3.9.Final.jar:/home/sid/.m2/repository/com/mchange/c3p0/0.9.2.1/c3p0-0.9.2.1.jar:/home/sid/.m2/repository/com/mchange/mchange-commons-java/0.2.3.4/mchange-commons-java-0.2.3.4.jar:/home/sid/.m2/repository/org/hibernate/hibernate-ehcache/4.3.9.Final/hibernate-ehcache-4.3.9.Final.jar:/home/sid/.m2/repository/org/hibernate/hibernate-search-orm/5.1.2.Final/hibernate-search-orm-5.1.2.Final.jar:/home/sid/.m2/repository/org/hibernate/hibernate-search-engine/5.1.2.Final/hibernate-search-engine-5.1.2.Final.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-analyzers-common/4.10.4/lucene-analyzers-common-4.10.4.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-facet/4.10.4/lucene-facet-4.10.4.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-queryparser/4.10.4/lucene-queryparser-4.10.4.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-core/4.10.4/lucene-core-4.10.4.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-sandbox/4.10.4/lucene-sandbox-4.10.4.jar:/home/sid/.m2/repository/org/apache/lucene/lucene-queries/4.10.4/lucene-queries-4.10.4.jar:/home/sid/.m2/repository/dom4j/dom4j/1.6.1/dom4j-1.6.1.jar:/home/sid/.m2/repository/org/slf4j/slf4j-api/1.6.0/slf4j-api-1.6.0.jar:/home/sid/.m2/repository/org/slf4j/jcl-over-slf4j/1.6.0/jcl-over-slf4j-1.6.0.jar:/home/sid/.m2/repository/org/slf4j/slf4j-log4j12/1.6.0/slf4j-log4j12-1.6.0.jar:/home/sid/.m2/repository/com/thoughtworks/xstream/xstream/1.4.3/xstream-1.4.3.jar:/home/sid/.m2/repository/xmlpull/xmlpull/1.1.3.1/xmlpull-1.1.3.1.jar:/home/sid/.m2/repository/xpp3/xpp3_min/1.1.4c/xpp3_min-1.1.4c.jar:/home/sid/.m2/repository/javax/mail/mail/1.4.1/mail-1.4.1.jar:/home/sid/.m2/repository/javax/activation/activation/1.1/activation-1.1.jar:/home/sid/.m2/repository/org/liquibase/liquibase-core/2.0.5/liquibase-core-2.0.5.jar:/home/sid/.m2/repository/org/openmrs/liquibase/ext/modify-column/2.0.2/modify-column-2.0.2.jar:/home/sid/.m2/repository/org/openmrs/liquibase/ext/identity-insert/1.2.1/identity-insert-1.2.1.jar:/home/sid/.m2/repository/org/openmrs/liquibase/ext/type-converter/1.0.1/type-converter-1.0.1.jar:/home/sid/.m2/repository/xerces/xercesImpl/2.8.0/xercesImpl-2.8.0.jar:/home/sid/.m2/repository/javax/validation/validation-api/1.0.0.GA/validation-api-1.0.0.GA.jar:/home/sid/.m2/repository/org/hibernate/hibernate-validator/4.2.0.Final/hibernate-validator-4.2.0.Final.jar:/home/sid/.m2/repository/org/codehaus/jackson/jackson-core-asl/1.9.13/jackson-core-asl-1.9.13.jar:/home/sid/.m2/repository/org/codehaus/jackson/jackson-mapper-asl/1.9.13/jackson-mapper-asl-1.9.13.jar:/home/sid/.m2/repository/org/codehaus/groovy/groovy-all/2.4.6/groovy-all-2.4.6.jar:/home/sid/.m2/repository/net/sf/ehcache/ehcache/2.10.0/ehcache-2.10.0.jar:/home/sid/.m2/repository/org/openmrs/api/openmrs-api/2.0.5/openmrs-api-2.0.5-tests.jar:/home/sid/.m2/repository/org/springframework/spring-test/4.1.4.RELEASE/spring-test-4.1.4.RELEASE.jar:/home/sid/.m2/repository/com/h2database/h2/1.4.187/h2-1.4.187.jar:/home/sid/.m2/repository/org/databene/databene-benerator/0.5.9/databene-benerator-0.5.9.jar:/home/sid/.m2/repository/org/databene/databene-webdecs/0.4.9/databene-webdecs-0.4.9.jar:/home/sid/.m2/repository/org/apache/poi/poi/3.5-beta5/poi-3.5-beta5.jar:/home/sid/.m2/repository/org/freemarker/freemarker/2.3.9/freemarker-2.3.9.jar:/home/sid/.m2/repository/org/databene/databene-commons/0.4.9/databene-commons-0.4.9.jar:/home/sid/.m2/repository/org/databene/databene-gui/0.1.9/databene-gui-0.1.9.jar:/home/sid/.m2/repository/net/sourceforge/jtds/jtds/1.2.2/jtds-1.2.2.jar:/home/sid/.m2/repository/mysql/mysql-connector-java/5.1.6/mysql-connector-java-5.1.6.jar:/home/sid/.m2/repository/org/apache/derby/derbyclient/10.4.2.0/derbyclient-10.4.2.0.jar:/home/sid/.m2/repository/postgresql/postgresql/8.3-603.jdbc4/postgresql-8.3-603.jdbc4.jar:/home/sid/.m2/repository/org/dbunit/dbunit/2.4.7/dbunit-2.4.7.jar:/home/sid/.m2/repository/xmlunit/xmlunit/1.3/xmlunit-1.3.jar:/home/sid/.m2/repository/com/carrotsearch/junit-benchmarks/0.7.0/junit-benchmarks-0.7.0.jar:/home/sid/.m2/repository/junit/junit/4.13/junit-4.13.jar:/home/sid/.m2/repository/org/hamcrest/hamcrest-core/2.2/hamcrest-core-2.2.jar:/home/sid/.m2/repository/org/hamcrest/hamcrest/2.2/hamcrest-2.2.jar:/home/sid/.m2/repository/org/hamcrest/hamcrest-library/2.2/hamcrest-library-2.2.jar:/home/sid/.m2/repository/org/mockito/mockito-core/3.2.4/mockito-core-3.2.4.jar:/home/sid/.m2/repository/net/bytebuddy/byte-buddy/1.10.5/byte-buddy-1.10.5.jar:/home/sid/.m2/repository/net/bytebuddy/byte-buddy-agent/1.10.5/byte-buddy-agent-1.10.5.jar:/home/sid/.m2/repository/org/objenesis/objenesis/2.6/objenesis-2.6.jar:/home/sid/.m2/repository/org/powermock/powermock-module-junit4/2.0.4/powermock-module-junit4-2.0.4.jar:/home/sid/.m2/repository/org/powermock/powermock-module-junit4-common/2.0.4/powermock-module-junit4-common-2.0.4.jar:/home/sid/.m2/repository/org/powermock/powermock-reflect/2.0.4/powermock-reflect-2.0.4.jar:/home/sid/.m2/repository/org/powermock/powermock-core/2.0.4/powermock-core-2.0.4.jar:/home/sid/.m2/repository/org/powermock/powermock-api-mockito2/2.0.4/powermock-api-mockito2-2.0.4.jar:/home/sid/.m2/repository/org/powermock/powermock-api-support/2.0.4/powermock-api-support-2.0.4.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit4 org.openmrs.module.fhir2.api.dao.impl.FhirObservationDaoImplTest,searchForObs_shouldSearchForObsByValueQuantity
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.springframework.cglib.core.ReflectUtils$2 (file:/home/sid/.m2/repository/org/springframework/spring-core/4.1.4.RELEASE/spring-core-4.1.4.RELEASE.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of org.springframework.cglib.core.ReflectUtils$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Double (java.lang.String and java.lang.Double are in module java.base of loader 'bootstrap')

	at org.hibernate.type.descriptor.java.DoubleTypeDescriptor.unwrap(DoubleTypeDescriptor.java:36)
	at org.hibernate.type.descriptor.sql.DoubleTypeDescriptor$1.doBind(DoubleTypeDescriptor.java:63)
	at org.hibernate.type.descriptor.sql.BasicBinder.bind(BasicBinder.java:90)
	at org.hibernate.type.AbstractStandardBasicType.nullSafeSet(AbstractStandardBasicType.java:286)
	at org.hibernate.type.AbstractStandardBasicType.nullSafeSet(AbstractStandardBasicType.java:281)
	at org.hibernate.loader.Loader.bindPositionalParameters(Loader.java:1995)
	at org.hibernate.loader.Loader.bindParameterValues(Loader.java:1966)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:1901)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1862)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1839)
	at org.hibernate.loader.Loader.doQuery(Loader.java:910)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:355)
	at org.hibernate.loader.Loader.doList(Loader.java:2554)
	at org.hibernate.loader.Loader.doList(Loader.java:2540)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2370)
	at org.hibernate.loader.Loader.list(Loader.java:2365)
	at org.hibernate.loader.criteria.CriteriaLoader.list(CriteriaLoader.java:126)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1718)
	at org.hibernate.internal.CriteriaImpl.list(CriteriaImpl.java:380)
	at org.openmrs.module.fhir2.api.dao.impl.FhirObservationDaoImpl.searchForObservations(FhirObservationDaoImpl.java:77)
	at org.openmrs.module.fhir2.api.dao.impl.FhirObservationDaoImplTest.searchForObs_shouldSearchForObsByValueQuantity(FhirObservationDaoImplTest.java:467)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:73)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:82)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:73)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:217)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:230)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:58)


Process finished with exit code 255

```

Reply:
@ibacher Found the solution for this. Thanks for the help